src/ExportGenerators.mss

```diff
@@ -1364,6 +1363,20 @@ function GenerateStaffGroups (score, barnum) {
             libmei.AddAttribute(std, 'key.mode', 'minor');
         }
 
+        if (s.FullInstrumentName != null)
+	{
+            label = libmei.Label();
+	    libmei.SetText(label, s.FullInstrumentName);
+            libmei.AddChild(std, label);
+	}
+
+        if (s.ShortInstrumentName != null)
+	{
+            labelAbbr = libmei.LabelAbbr();
+	    libmei.SetText(label, s.ShortInstrumentName);
```

Comment:
Yes, of course! :man_facepalming:

Reply:
Now we have mixed tabs and spaces. (At least this makes me feel a tiny little bit less stupid about the mess *I'm* routinely making.)

Reply:
BTW, does any of you know how to modify such tiny things in others' pull requests? Seems like unnecessary work. I could of course just change things and push them directly instead of merging this pull request, but I think we've seen that it makes sense to not directly push and have other people have a look first. I could also create a new pull request, but that's messy as well.

Reply:
Using tabs / spaces produces in my editors the same look. Is there a formatter somewhere for this? At least, this should now be fine.

Reply:
In your review, you can propose a change directly in the reviewed code. 

When you click on the plus sign next to the corresponding code line, the following editor has a new icon that looks like file sheets with a + and - on it (next to the `AA`, `B`, and `i` icon).

Clicking on this icon will copy the existing code into the editor window, and you can make your proposed changes (cf. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request nr. 6). This is also possible for multiple lines, then you would have to select all the lines you need beforehand (see ibidem, nr. 4). The PR author can then accept these changes and they would be commited (one by one or as a batch job) to the PR directly. (cf. https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request )

Hope this helps.